CONTRIBUTING.md

```diff
@@ -35,6 +35,30 @@ proposals against the OCaml distribution. (Code changes, but also
 improvement to documentation or implementation comments, which are
 valuable changes on their own.)
 
+## Workflow
+
+All changes to the OCaml distribution need to be processed through the
+GitHub Pull Request (PR) system.  In order to propose a change, a
+contributor thus needs to have a GitHub account, fork the ocaml/ocaml
+repository, create a branch for the proposal on their fork and submit
+it as a Pull Request on the upstream repository.  (If you are not yet
+familiar with GitHub, don't worry, all these steps are actually quite
+easy!)
```

Comment:
Actually, I cannot, I'd need to open a new PR for that, and have someone review it 😄  

More seriously, I don't think the old workflow could make sense for tiny patches that could easily be applied directly by any maintainer.  But now that one needs to go through a PR anyway, let's document the constraint.  It's not to say that one should necessarily reject a contribution coming as a patch, but we don't need to document it, I think.